src/runner/powertoy_module.cpp

```diff
@@ -20,7 +20,7 @@ PowertoyModule load_powertoy(const std::wstring_view filename)
     if (!module)
     {
         FreeLibrary(handle);
-        winrt::throw_last_error();
+        winrt::throw_hresult(winrt::hresult(E_POINTER));

```

Comment:
This exception is caught by https://github.com/microsoft/PowerToys/blob/85c0eaa5980cabab16089fea3a2c6f148242a368/src/runner/main.cpp#L132-L139
that doesn't report any error, we should add a messagebox there reporting the module name that failed.

Reply:
Resolved.